.github/workflows/test.yaml

```diff
@@ -385,7 +479,7 @@ jobs:
         path: "${{ github.workspace }}/src/github.com/snapcore/snapd/.coverage/coverage*.cov"
 
   code-coverage:
-    needs: [unit-tests]
+    needs: [unit-tests, unit-tests-special]
```

Comment:
some build tags trigger execution of additional code which is useful to include in coverage

Reply:
How are the coverage results merged? Asking because Sergio mentioned that this will be send to the TIOBI server as well, so checking to make sure the various uploads yield expected end result.

Reply:
The coverage is merged by  `codecov/codecov-action@v3`, I don't know the exact details, but we just provide the coverage file artifacts like before.